### ISSUE TITLE
Debugger in VS show null reference exception for OperationOutcome Issue if details are null.

### ISSUE THREAD

**Issue by Ivanidzo4ka** [FirelyTeam/firely-net-sdk#1529]
**Describe the bug**
I've create OperationOutcome object with issue in it. 

According to spec: https://www.hl7.org/fhir/operationoutcome.html  
Details is something what can be absent, at least this is how I interpret 0..1, so I omit adding details for my issue.

I find it annoying to see null exception in my watch window 
![image](https://user-images.githubusercontent.com/1523833/97916605-138ec680-1d08-11eb-97b9-d0a5177f9927.png)

**To Reproduce**
```
var outcome = new OperationOutcome
                {
                    Id ="",
                    Issue = new List<OperationOutcome.IssueComponent>
                    {
                        new OperationOutcome.IssueComponent
                        {
                            Severity = OperationOutcome.IssueSeverity.Error,
                            Code = issueType,
                            Diagnostics = diagnosticInfo,
                        },
                    },
```
hover over outcome and unroll Issue list. Observe null exception instead of proper display.

**Expected behavior**
I would like to not see null exception, and see at least code, if details are not present.

**Screenshots**
Above

**Version used:**
 - FHIR Version: [ STU3]
 - Version: [e.g. 1.9.0]
But I'm sure it's applicable across all version.

**Additional context**
https://github.com/FirelyTeam/fhir-net-api/blob/add8073b8d096f66cde13f54c940048ba4363e0a/src/Hl7.Fhir.Core/Model/OperationOutcome.cs#L122
I think all is needed is null check for _Details field in that line.

**Comment by marcovisserFurore** [comment]
Hi Ivan,

Thanks for your issue.

We can change line 122 in `OperationOutcome.cs` into this:
```c#
 return String.Format("Code=\"{0}\" {1}", this.Code, Details?.DebuggerDisplay("Details.") ?? "(no details)");
```

**Comment by marcovisserFurore** [comment]
Solved in 2.0-final
